suite-common/bluetooth/src/bluetoothReducer.ts

```diff
@@ -131,6 +142,43 @@ export const prepareBluetoothReducerCreator = <T extends BluetoothDeviceCommon>(
                         );
                         if (foundKnownDevice === undefined) {
                             state.knownDevices.push(device);
+                        } else {
+                            delete state.autoConnectPolicy[bluetoothProps.id];
+                        }
+                    }
+                },
+            )
+            .addMatcher(
+                action => action.type === deviceActions.deviceDisconnect.type,
+                (state, { payload }: ReturnType<typeof deviceActions.deviceDisconnect>) => {
+                    const id = payload.bluetoothProps?.id;
+                    const affectedDevice = id && state.knownDevices.find(it => it.id === id);
+                    if (
+                        affectedDevice &&
+                        state.autoConnectPolicy[id]?.type !== 'autoconnect-disabled'
+                    ) {
+                        state.autoConnectPolicy[id] = {
+                            type: 'recently-disconnected',
+                            timestamp: Date.now() as Timestamp,
+                        };
+                    }
+                },
+            )
+            .addMatcher(
+                action => action.type === deviceActions.devicePushNotification.type,
+                (
+                    state,
+                    {
+                        payload: { device, type },
+                    }: ReturnType<typeof deviceActions.devicePushNotification>,
+                ) => {
+                    if (type === 3) {
```

Comment:
What is magic number? Do we have a named constant for it?

Reply:
missing enum export from trezor-connect